### PR TITLE
docker image version changed to latest.

### PR DESCRIPTION
**Pull request by sujithmn**

### CODE REVIEW

File: .github/workflows/buildandpush.yml [hunk 1]
```diff
@@ -48,9 +48,9 @@ jobs:
                
           echo VERSION=$VERSION
           
-          docker tag $JAVA_IMAGE_NAME $JAVA_IMAGE_ID:${{ github.sha }}
-          docker push $JAVA_IMAGE_ID:${{ github.sha }}
+          docker tag $JAVA_IMAGE_NAME $JAVA_IMAGE_ID
```
Comment (ssarmokadam) [inline .github/workflows/buildandpush.yml:51]: Can you remove tagging while building docker image?

File: .github/workflows/buildandpush.yml [hunk 2]
```diff
@@ -48,9 +48,9 @@ jobs:
...
+          docker push $JAVA_IMAGE_ID
 
-          docker tag $ANGULAR_IMAGE_NAME $ANGULAR_IMAGE_ID:${{ github.sha }}
-          docker push $ANGULAR_IMAGE_ID:${{ github.sha }}
+          docker tag $ANGULAR_IMAGE_NAME $ANGULAR_IMAGE_ID
```
Comment (ssarmokadam) [inline .github/workflows/buildandpush.yml:54]: same as above

File: .github/workflows/buildandpush.yml (hunk 1 above)
Comment (sujith-mn) [reply .github/workflows/buildandpush.yml:40]: Tagging is required to push the image.

File: .github/workflows/buildandpush.yml (hunk 2 above)
Comment (sujith-mn) [reply .github/workflows/buildandpush.yml:43]: Tagging is required to push the image.

File: .github/workflows/buildandpush.yml (hunk 1 above)
Comment (ssarmokadam) [reply .github/workflows/buildandpush.yml:40]: I am referring to docker build . --file java/Dockerfile.githubactions --tag $JAVA_IMAGE_NAME --label "runnumber=${GITHUB_RUN_ID}" but was not able to add comment thr

**Review by ssarmokadam — APPROVED** [review]

**Pull request merged**